server/routes/movePrisoner/prisonerSelect.ts

```diff
@@ -33,7 +33,11 @@ export default class PrisonerSelectRoutes {
       user
     )
 
-    return this.renderView(req, res, { searchResults, searchTerm })
+    const availablePrisoners = searchResults
+      .filter(result => result.indeterminateSentence || result.conditionalReleaseDate > new Date())
```

Comment:
bit naff to have logic in two places - could you extract out filter function and re-use?

Reply:
I was thinking Martin Fowler's rule of 3 😁. But fair enough, probably better to have their own tests anyway.